pages/partners/[partner].vue

```diff
@@ -0,0 +1,258 @@
+                    v-for="desc in description"
+                  >
+                    {{ desc }}
+                  </p>
+                  <nuxt-link to="/contact">
+                    <v-btn variant="text" style="padding: 0px">
+                      <span class="text-[#ff7b02]">Contact Us To Learn More</span>
+                      <v-icon right class="ml-2 text-[#ff7b02]">mdi-arrow-right</v-icon>
+                    </v-btn>
+                  </nuxt-link>
+                </div>
+              </v-window-item>
+              <v-window-item value="products">
+                <div class="p-5 md:p-8" style="margin: 0px auto; padding-top: 1.5rem">
+                  <!-- PRODUCTS WEB VIEW -->
+                  <div class="grid hidden-sm-and-down" :class="getColsCount()">
+                    <!-- copying what's in the current website -->
+                    <v-card flat class="mx-4 xl:mx-3" v-for="product in products">
+                      <v-img
+                        :src="`/static/images/${product.image}`"
+                        :lazy-src="`/static/images/${product.image}`"
+                        class="floating-image"
+                      ></v-img>
+                      <v-card-text
+                        class="text-xs md:text-sm xl:text-md 2xl:text-lg font-Poppins font-weight-light"
+                        style="padding: 25px 5px"
+                      >
+                        <!-- md:text-xs 2xl:text-base -->
+                        {{ product.text }}
+                      </v-card-text>
+                    </v-card>
+                  </div>
+
+                  <!-- PRODUCTS MOBILE VIEW -->
+                  <div class="grid hidden-md-and-up">
+                    <!-- copying what's in the current website -->
+                    <v-carousel cycle hide-delimiters touch show-arrows="hover" progress="#ff7b02">
+                      <v-carousel-item v-for="product in products">
+                        <v-card flat class="mx-4 xl:mx-3">
+                          <v-img
+                            :src="`/static/images/${product.image}`"
+                            :lazy-src="`/static/images/${product.image}`"
+                            class="floating-image"
+                          ></v-img>
+                          <v-card-text class="text-xs font-Poppins" style="padding: 25px 0px">
+                            {{ product.text }}
+                          </v-card-text>
+                        </v-card>
+                      </v-carousel-item>
+                    </v-carousel>
+                  </div>
+                </div>
+              </v-window-item>
+            </v-window>
+          </v-col>
+        </v-row>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup>
+import { ref, onMounted } from 'vue';
+import { useRoute } from 'vue-router';
+import { partnersData } from '../../data/partners-data.ts';
+
+useHead({
+  title: 'DataStax',
```

Comment:
use a dynamic text here. it will always be DataStax

Reply:
![image](https://github.com/BrieQiel/sctc-website-master/assets/87899638/1b2d518c-d918-4e80-857a-08401b2b530a)

placing it in mounted would be the wei